### PR TITLE
Add support for admins config property, remove edge tag

### PR DESCRIPTION
**Pull request by sdsznsk**
When merged, this PR will:
- title

The edge tag I assume was a leftover after transition so might as well I guess.

### CODE REVIEW

File: launch.py [hunk 1]
```diff
@@ -79,6 +79,12 @@ def bool_str(text):
         adminPassword = random_passphrase()
         config["game"]["passwordAdmin"] = adminPassword
         print(f"Admin password: {adminPassword}")
+    if env_defined("GAME_ADMINS"):
+        config["game"]["admins"] = []
+        admins = str(os.environ["GAME_ADMINS"]).split(",")
+        admins[:] = [admin for admin in admins if admin]  # Remove empty items form list
+        for admin in admins:
+            config["game"]["admins"].append(admin)
```
Comment (jonpas) [inline launch.py:87]: Why not just?
```suggestion
        config["game"]["admins"] = admins
```

Comment (sdsznsk) [reply launch.py:87]: Drank too much, true

File: launch.py [hunk 2]
```diff
@@ -79,6 +79,11 @@ def bool_str(text):
         adminPassword = random_passphrase()
         config["game"]["passwordAdmin"] = adminPassword
         print(f"Admin password: {adminPassword}")
+    if env_defined("GAME_ADMINS"):
+        config["game"]["admins"] = []
```
Comment (jonpas) [inline launch.py:83]:
```suggestion
```
This too then. :P

Comment (sdsznsk) [reply launch.py:83]: True

**Review by jonpas — APPROVED** [review]

**Pull request merged**